themes/digital.gov/src/scss/new/guides/_guide-menu-bar.scss

```diff
@@ -78,3 +78,101 @@
     }
   }
 }
+
+.dg-accordion-mobile-menu {
```

Comment:
The `dg-accordion-mobile-menu` overrides the `usa-accordion` component styles to work with the guide menu icons and nav list.